imls-backend/db/migrations/20220831042321_bin_devices_over_time.sql

```diff
@@ -0,0 +1,61 @@
+-- migrate:up
```

Comment:
because of the date numbering schema, this migration will run _before_ the last migration currently in `main` which is `20220907165121_modify_bin_devices_per_hour.sql`. Not sure if this breaks anything, but just something to keep in mind.

Reply:
This one should be fine as long as it runs before 20220902170318, which applies the update to the function.